lib/iris/tests/unit/fileformats/netcdf/test_Saver.py

```diff
@@ -180,7 +181,7 @@ def test_reserved_attributes(self):
             self.assertEqual(res, 'something something_else')
 
 
-class TestCoordSystems(tests.IrisTest):
+class Test_create_cf_grid_mapping(tests.IrisTest):
```

Comment:
:+1: for updating the name - thanks.

Please could you add a second underscore at the start, i.e. `Test__create_cf...`